src/components/@shared/FormFields/FilesInput/Info.module.css

```diff
@@ -31,6 +31,10 @@
   padding-right: calc(var(--spacer) / 2);
 }
 
+.hideUrl {
+  filter: blur(0.2rem);
```

Comment:
What is the purpose of this blurring?

<img width="581" alt="Screen Shot 2022-10-14 at 13 44 01" src="https://user-images.githubusercontent.com/90316/195849895-33c4fdd0-f6b8-466d-8781-fa31daa6714c.png">

Either we prevent access to that URL completely (I can just select and copy it, I can see it in the source code, etc.) or we show it. This in between does not solve anything I think. And on top of that, every new file I add gets blurred too. It is always fine showing that URL to the actual publisher.

Reply:
aaah sorry, I get it now, was already wondering and started inspecting how the heck we get that existing URL decrypted :-) 

Still, when adding a new file, we have to show the just added URL

Reply:
yea, i was a bit confused about the blur, not sure what the purpose it.

Reply:
the blur is used to hide the fact that we can't access the url of the asset. @kremalicious is right, we should display the blur only when the user access the edit form, but if he edits the fields, the new url should be displayed without blur.